src/libsystemd/sd-network/network-util.c

```diff
@@ -103,3 +104,24 @@ int parse_operational_state_range(const char *str, LinkOperationalStateRange *ou
 
         return 0;
 }
+
+char *link_get_type_string(unsigned short iftype, sd_device *device) {
```

Comment:
i find the order of args strange btw, i think the order should be the other way round